src/classes/BDI_DataImportService.cls

```diff
@@ -1777,21 +1777,24 @@ global with sharing class BDI_DataImportService {
                 addr.Address_Type__c = 'Home';
 
                 if (con1 != null) {
-                    ADDR_Addresses_TDTM.copyAddressAddrSObj(addr, con1, 'Mailing', 'npe01__Primary_Address_Type__c');
+                    NPSP_Address address = new NPSP_Address(addr);
+                    address.copyOntoSObject(con1, 'Mailing','npe01__Primary_Address_Type__c');
```

Comment:
Eventually this should be: `npspContact.mailingAddress(address);`, instead of telling the Address to copy itself onto the contact, we will tell the contact to use the Address.